ProcessLib/HeatTransportBHE/BHE/BHE_CXA.h

```diff
@@ -0,0 +1,151 @@
+/**
```

Comment:
Parts of this class looks very similar to parts of the class `BHE_CXC`. Maybe `BHE_CXA` and `BHE_CXC` can be derived from a common base class?

Reply:
Fixed. The similar parts were removed to a common base class.